pkg/safeguards/helpers.go

```diff
@@ -111,11 +111,11 @@ func IsYAML(path string) bool {
 }
 
 // GetManifestFiles uses filepath.Walk to retrieve a list of the manifest files within the given manifest path
-func GetManifestFiles(p string) ([]c.ManifestFile, error) {
-	var manifestFiles []c.ManifestFile
+func GetManifestFiles(p string) ([]types.ManifestFile, error) {
```

Comment:
This function is unused, you can remove it

Reply:
Seemed to have duplicated it, removed now.